sql/moz-fx-data-marketing-prod/ga_derived/www_site_hits_v2/query.sql

```diff
@@ -117,7 +117,14 @@ get_all_events_in_each_session AS (
     SPLIT(a.page_location, '?')[OFFSET(0)] AS page_location,
     a.is_entrance,
     DENSE_RANK() OVER (PARTITION BY visit_identifier ORDER BY event_timestamp ASC) AS hit_number,
-    ROW_NUMBER() OVER (PARTITION BY visit_identifier ORDER BY event_timestamp ASC) AS row_nbr
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        visit_identifier
+      ORDER BY
+        event_timestamp,
+        event_name,
+        page_location ASC
```

Comment:
no `event_index` of some sort in the event params?

Reply:
I wish 😭